docs/source/guide/secrets-manager.rst

```diff
@@ -0,0 +1,85 @@
+.. Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+   This work is licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 4.0
+   International License (the "License"). You may not use this file except in compliance with the
+   License. A copy of the License is located at http://creativecommons.org/licenses/by-nc-sa/4.0/.
+
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+   either express or implied. See the License for the specific language governing permissions and
+   limitations under the License.
+   
+.. _aws-boto3-secrets-manager:   
+
+###############################################
+Retrieving a Secret from AWS Secrets Manager

```

Comment:
Let's make the page title just `Secrets Manager` and move this title down as this page title is what shows up on the side navigation bar.

Reply:
For reference this is what it currently looks like:

<img width="300" alt="screen shot 2018-08-30 at 3 11 10 pm" src="https://user-images.githubusercontent.com/1935727/44882151-fa40ea00-ac66-11e8-96b3-e9a7c99dda61.png">